cmd/policy-main.go

```diff
@@ -394,18 +421,23 @@ func runPolicyLinksCmd(args cli.Args, recursive bool) {
 func runPolicyCmd(args cli.Args) {
 	var operation, policyStr string
 	var probeErr *probe.Error
-	perms := accessPerms(args.Get(0))
-	targetURL := args.Get(1)
+	perms := accessPerms(args.Get(1))
+	targetURL := args.Get(2)
 	if perms.isValidAccessPERM() {
 		probeErr = doSetAccess(targetURL, perms)
 		operation = "set"
 	} else if perms.isValidAccessFile() {
 		probeErr = doSetAccessJSON(targetURL, perms)
-		operation = "setJSON"
+		operation = "set json"
```

Comment:
is this `set-json` again? The operation name can also be set-json as well. IMO

Reply:
Updated it to `set-json` only .